### ISSUE TITLE
Is there a way to internationalize json domain titles ?

### ISSUE THREAD

**Issue by rperrot** [jbms/sphinx-immaterial#274]
Hi, 

first of all, thank you for your really helpful theme/extension.

I' trying to use json domain in a non english document and I'm stuck with the "required/optional" members string. I'd like to change it but I can't find a way to change it using a parameter/option.

I think the issue is because theses strings are hard coded here : 

https://github.com/jbms/sphinx-immaterial/blob/9adb3f18bf759b287627b0dcaff7e1bd512ed0b6/sphinx_immaterial/apidoc/json/domain.py#L866

Is there a way to customize it or it's impossible ?

Yours,

**Comment by 2bndy5** [comment]
True, the strings are hardcoded in English, and there is no config option to change or remove them. We could use [Sphinx' own translation mechanism](https://www.sphinx-doc.org/en/master/usage/advanced/intl.html), but I think that would require a .po file (in the doc project's source) that defines the translation for the ~~words~~ phrases "Required members" and "Optional members". This idea would be simple to implement on our end:
```python
                heading = "%s members" % ("Required" if required else "Optional") 
                 field_list, body = self._make_field(_(heading))
```
But it will require a message catalog (.po file) of translations on the user's end.

**Comment by 2bndy5** [comment]
The solution for this was released in [v0.12.0](https://pypi.org/project/sphinx-immaterial/0.12.0/)
